doc/release-notes.rst

```diff
@@ -2,6 +2,123 @@
  Release Notes
 ===============
 
+v12.2.1 Luminous
+================
+
+This is the first bugfix release of Luminous v12.2.x long term stable
+release series. We recommend all the users of 12.2.x series update.
```

Comment:
Can we model this paragraph on the analogous paragraph from the v10.2.1 release notes?

```
This is the first bugfix release for Jewel.  It contains several annoying
packaging and init system fixes and a range of important bugfixes across
RBD, RGW, and CephFS.

We recommend that all v10.2.x users upgrade.

For more detailed information, see :download:`the complete changelog <changelog/v10.2.1.txt>`.
```